cmake/Modules/LocalRav1e.cmake

```diff
@@ -1,5 +1,5 @@
 set(AVIF_LOCAL_RAV1E_GIT_TAG v0.7.1)
-set(AVIF_LOCAL_CORROSION_GIT_TAG v0.4.10)
+set(AVIF_LOCAL_CORROSION_GIT_TAG v0.5.0)
```

Comment:
I tried updating cargo-c to the latest v0.10.0 release, but there was a build failure on Windows. So I removed that change from this pull request.